woocommerce/woocommerce-ios
# [Blaze] Validate campaign image dimensions

Closes: #12463


## Description


Blaze backend expects the campaign image size to be greater than or equal to 500*500.

To inform users about this requirement, we are validating the selected campaign image dimensions and showing a local error alert about the expected image dimensions. 

Internal discussion in which we decided to follow the above approach - p1712745183124909/1712674079.063239-slack-C03L1NF1EA3

## Testing instructions


Prerequisites
- Woo Store eligible for Blaze
- A published product with an image with dimensions less than 500*500. 
- An image with dimensions less than 500*500 in your device/simulator library.

Steps
- Login into a Woo store eligible for Blaze.
- Navigate to the Products tab and tap on the product with the image with dimensions less than 500*500. 
- You should see the "Promote with Blaze" button.
- Tap "Promote with Blaze" -> "Confirm Details". 
- You should see an error alert asking you to select an image with minimum dimensions of 500*500.
- Tap "Edit ad" and navigate to the edit screen.
- Tap "Change image" and choose an image with dimensions less than 500*500. 
- You should see an error alert asking you to select an image with minimum dimensions of 500*500.

## Screenshots


#### Creation form - When a product image does not have 500 * 500

https://github.com/woocommerce/woocommerce-ios/assets/524475/6fa001b9-3719-48d8-b3e3-c12e239be92e

#### Edit image form - When selected image 500 * 500

https://github.com/woocommerce/woocommerce-ios/assets/524475/18ebacb4-68e6-4a2e-ac36-c26f7771ec1a


---
